### PR TITLE
[milestone/12.2] Milestone 12.2

### PR DESCRIPTION
**Pull request by felix-schwarz**
## Description
Release 12.2 PR

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] Added an issue with details about all relevant changes in the [**iOS documentation repository**](https://github.com/owncloud/docs-client-ios-app/issues).
- [ ] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] Added changelog files for the fixed issues in folder changelog/unreleased

### CODE REVIEW

**Comment by felix-schwarz** [comment]
@hosy @jesmrec Everything is merged into `milestone/12.2` now. Please check and let me know once you have approved it. I can then (rebase) merge the changes into `master` and create a tag for the release.

**Review by jesmrec — APPROVED** [review]

**Comment by hosy** [comment]
@jesmrec @felix-schwarz Before approving this PR: I think we should include the #1347 into this release.

**Comment by felix-schwarz** [comment]
@hosy Since this release already is delayed, I'd prefer to ship 2.2 now - and include #1347 either in a quick, subsequent 2.2.1 or 2.3 as soon as it has passed QA.

Otherwise, we'd need to wait for QA of #1347 and then go through QA for the merged result again, delaying the release further.

**Review by hosy — APPROVED** [review]

**Pull request merged**
